src/modules/formBuilder/components/itemEditor/conditionals/InitialValue.tsx

```diff
@@ -74,54 +94,80 @@ const InitialValue: React.FC<Props> = ({ itemType, control }) => {
+                      name='initial.0.valueBoolean'
+                      control={control}
+                      shouldUnregister
+                      render={({
+                        field: { ref, ...field },
+                        fieldState: { error },
+                      }) => (
+                        <YesNoRadio
+                          label='Initial Value (Boolean)'
+                          error={!!error}
+                          helperText={error?.message}
+                          {...field}
+                        />
+                      )}
+                    />
+                  )}
+                  {valueField === 'valueCode' && (
+                    <>
+                      {pickList && pickList.length > 0 ? (
+                        <ControlledSelect
+                          name='initial.0.valueCode'
+                          control={control}
+                          label='Initial Value'
+                          options={pickList}
+                          placeholder='Select an option'
+                        />
+                      ) : (
+                        <ControlledTextInput
+                          name='initial.0.valueCode'
+                          control={control}
+                          label='Initial Value'
+                        />
+                      )}
+                    </>
+                  )}
+                </>
               )}
-              {pickList && pickList.length > 0 ? (
+              {advanced.localConstant && (
                 <ControlledSelect
-                  name='initial.0.valueCode'
-                  control={control}
-                  label='Initial Value (Pick List)'
-                  options={pickList}
-                  placeholder='Select an option'
-                />
-              ) : (
-                <ControlledTextInput
-                  name='initial.0.valueCode'
+                  name='initial.0.valueLocalConstant'
                   control={control}
-                  label='Initial Value'
+                  label='Local Constant'
+                  placeholder='Select local constant'
+                  options={localConstantsPickList}
+                  helperText='Set initial value based on a local constant'
                 />
               )}
-              <ControlledSelect
-                name='initial.0.valueLocalConstant'
-                control={control}
-                label='Local Constant'
-                placeholder='Select local constant'
-                options={localConstantsPickList}
-                helperText='Set initial value based on a local constant'
-              />
+              <Box sx={{ mt: 2 }}>
+                <Typography typography='body2' fontWeight={600}>
+                  Advanced Options
+                </Typography>
+                <Stack>
+                  <LabeledCheckbox
+                    label='Initialize using a Local Constant'
```

Comment:
I wonder if "initialize" might be a bit jargon-y for our users, though maybe this is ok because it's an advanced setting anyway. Could this be 'Set initial value based on a local constant', like the helper text above?